src/Confluent.SchemaRegistry.Serdes.Json/JsonSerializerConfig.cs

```diff
@@ -64,13 +64,22 @@ public static class PropertyNames
             /// <summary>
             ///     Specifies whether or not the JSON serializer should use the latest subject
             ///     version for serialization.
-            ///     WARNING: There is no check that the latest schema is backwards compatible
-            ///     with the schema of the object being serialized.
+            ///     WARNING: There is no check that the latest schema is compatible
+            ///     with the schema of the object being serialized by default.
+            ///     Use the LatestCompatibilityStrict config property to enable this.
             ///
             ///     default: false
             /// </summary>
             public const string UseLatestVersion = "json.serializer.use.latest.version";
 
+            /// <summary>
+            ///     Specifies whether or not the JSON serializer should check the compatibility 
+            ///     with the latest schema of the subject if use.latest.version is set to true.
+            ///
+            ///     default: false
+            /// </summary>
+            public const string LatestCompatStrict = "json.serializer.latest.compatibility.strict";
```

Comment:
the public property name should match the config name `LatestCompatibilityStrict`

Reply:
that is a just a local variable name which doesn't matter at all - the exposed name is `latest.compatibility.strict`, the property name should match this.